setup.sh

```diff
@@ -2,5 +2,6 @@ sudo apt-get update
 
 sudo apt-get -y install libffi-dev
 sudo apt-get -y install libssl-dev
-sudo apt-get -y install python-tox
+sudo apt-get -y install python-pip
```

Comment:
Can you explain this change? What is the problem with installing tox via apt-get?

Reply:
What do you think we'll only consider the files with relevant changes to the Mesos plugin? There are many files that do not make sense to change. This is an example. @tellesnobrega @armstrongmsg  @iurygregory @igornatanael

Reply:
It makes perfect sense to only change things related to mesos on this pull request

Reply:
Done.